### ISSUE TITLE
#bug # accessibility.the translate and reactions butten are not accessible for screenreaders.

### ISSUE THREAD

**Issue by alireza1385mamani** [UnigramDev/Unigram#2616]
**Describe the bug**
the translate and reactions butten are not accessible for screenreaders.
I mean that there should be a a butten that is accessible with screen readers to add reactions and translate something.
**To Reproduce**
**Expected behavior**
**Screenshots**
**Version Info**
 - Unigram: [e.g. Unigram 8.4 (6853) X64

 - Windows: [e.g. October 2018 update]
*Windows 11 21H2 (x64) build 22000.376
**Additional context**
I'm using nvda screen reader version 2021.3.1

**Comment by FrayxRulez** [comment]
I just replied to you in the other issue that you just created:
- You must enable Translation from Settings > Languages 
- Reactions aren't currently supported

**Comment by alireza1385mamani** [comment]
@FrayxRulez I've done what you said. but there isn't any translate butten.
